cli.js

```diff
@@ -1,6 +1,19 @@
 #!/usr/bin/env node
 'use strict';
 
+// Prefer the local installation of AVA.
+var resolveFrom = require('resolve-from');
+var localCLI;
+try {
+	localCLI = resolveFrom('.', 'ava/cli');
+} catch (e) {}
+
+if (localCLI && localCLI !== __filename) {
+	console.warn('Using local install of AVA.');
```

Comment:
OK. Honestly, that line was there to appease you: https://github.com/sindresorhus/xo/pull/32#issuecomment-155032045.

I thought you brought up a good point, that silently changing standard behavior is a bad thing.

@sindresorhus, what do you think?